Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:latest
+FROM golang:1.17
```

Comment:
Maybe use `golang:1.18` here, keep it consistent with `go.mod`.

Reply:
sure!